odoo/addons/base/tests/test_ir_actions.py

```diff
@@ -383,3 +383,63 @@ def test_unlink_with_dependant(self):
         # uninstall mode: unlink dependant fields
         field.with_context(_force_unlink=True).unlink()
         self.assertFalse(dependant.exists())
+
+    def test_related_field(self):
+        """ create a related field, check filled values"""
+        SOURCE_MODEL = 'res.users'
+        SOURCE_FIELD = 'login'
+
+        TARGET_MODEL = 'res.partner'
+        target_model = self.env['ir.model'].search([('model', '=', TARGET_MODEL)])
+
+        target_records = self.env[TARGET_MODEL]
+        source_records = self.env[SOURCE_MODEL]
+        for i in range(10):
+            # create a user
+            source = self.env[SOURCE_MODEL].create({
+                'name': 'salesperson %s' % i,
+                SOURCE_FIELD: 'value_%s' % i,
+            })
+            # create partner with user_id (Salesperson) field
+            target = self.env[TARGET_MODEL].create({
+                'name': '%s' % i,
+                'user_id': source.id,
+            })
+            source_records |= source
+            target_records |= target
+
+        if odoo.release.version_info[0] >= 13:
+            source_records.flush()
+            target_records.flush()
```

Comment:
Please don't do that.  Instead, modify the forward-port in 13.0, and the rest will follow.
If a test is not obvious, it may be buggy...

Reply:
I replaced it with a comment